src/gui_common/dialogs/CustomDialog.cs

```diff
@@ -311,14 +315,19 @@ public override void _Draw()
         DrawString(titleFont, titlePosition, translatedWindowTitle, titleColor,
             (int)(RectSize.x - customPanel.GetMinimumSize().x));
 
+        var closeButtonRect = closeButton!.GetRect();
+
+        // Draw close button
+        DrawTextureRect(closeButtonTexture, closeButtonRect, false, closeButtonColor);
```

Comment:
Actually could you link the relevant issue here and briefly explain that this button is rendered in a custom way due to a bug with it otherwise disappearing sometimes?